test/simulator/timestamp/src/include/connection_simulator.h

```diff
@@ -55,4 +74,14 @@ class connection_simulator {
     /* Member variables */
     private:
     std::vector<session_simulator *> _session_list;
+    enum system_timestamps { oldest_timestamp, stable_timestamp, durable_timestamp };
+    std::map<std::string, system_timestamps> system_timestamps_map;
+    /*
+     * Timestamp manager is responsible for validating all the timestamps, including the global
+     * timestamps; stable, oldest and durable in the connection.
+     */
+    timestamp_manager *ts_mgr;
+    uint64_t oldest_ts;
```

Comment:
All three could be declared in one line.

Reply:
Is that something we do in classes?

Reply:
Hmm, I see it is declared in a separate line in the CPP suite so I guess It's entirely your preference.